source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -998,6 +998,337 @@ See the `Load Balancer Specification <../load-balancers/load-balancers.rst#event
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 1
+
+   * - Key
+     - Suggested Type
+     - Value
+
+   * - serverHost
+     - String
+     -  the hostname or IP address for the endpoint the pool is for.
+
+   * - serverPort
+     - Int
+     - The port for the endpoint the pool is for. Optional; only present if a port was specified.
+
+Pool Created Message
+---------------------
+In addition to the common fields defined above, this message MUST contain the following key-value pairs:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 1
+
+   * - Key
+     - Suggested Type
+     - Value
+
+   * - message
+     - String
+     - "Connection pool created"
+
+   * - maxIdleTimeMS
+     - Int
+     - The maxIdleTimeMS value for this pool. Optional; only required to include if the user specified a value.
+
+   * - minPoolSize
+     - Int
+     - The minPoolSize value for this pool. Optional; only required to include if the user specified a value.
+
+   * - maxPoolSize
+     - Int
+     - The maxPoolSize value for this pool. Optional; only required to include if the user specified a value.
+
+   * - maxConnecting
+     - Int
+     - The maxConnecting value for this pool. Optional; only required to include if the driver supports this option and the user
+       specified a value.
+
+   * - waitQueueTimeoutMS
+     - Int
+     - The waitQueueTimeoutMS value for this pool. Optional; only required to include if the driver supports this option and the
+       user specified a value.
+
+   * - waitQueueSize
+     - Int
+     - The waitQueueSize value for this pool. Optional; only required to include if the driver supports this option and the
+       user specified a value.          
+
+   * - waitQueueMultiple
+     - Int
+     - The waitQueueMultiple value for this pool. Optional; only required to include if the driver supports this option and the
+       user specified a value.
+
+The unstructured form SHOULD be as follows, using the values defined in the structured format above to fill in placeholders as appropriate:
+
+  Connection pool created for host with hostname {{serverHost}} on port {{serverPort}} using options maxIdleTimeMS={{maxIdleTimeMS}},
+  minPoolSize={{minPoolSize}}, maxPoolSize={{maxPoolSize}}, maxConnecting={{maxConnecting}}, waitQueueTimeoutMS={{waitQueueTimeoutMS}},
+  waitQueueSize={{waitQueueSize}}, waitQueueMultiple={{waitQueueMultiple}}
+
+Pool Ready Message
+------------------
+In addition to the common fields defined above, this message MUST contain the following key-value pairs:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 1
+
+   * - Key
+     - Suggested Type
+     - Value
+
+   * - message
+     - String
+     - "Connection pool ready"
+
+The unstructured form SHOULD be as follows, using the values defined in the structured format above to fill in placeholders as appropriate:
+
+  Connection pool ready for host with hostname {{serverHost}} on port {{serverPort}}
```

Comment:
Even if we do keep separate structured fields for the hostname and port portion, I do think this change is worth making, since otherwise, I'd imagine any consumers of these logs would need to write custom logic to parse "for host with hostname" and "on port", whereas "hostname:port" probably has many existing parsers. Similarly, grepping for "a:1234" is a bit quicker than grepping for the whole english phrase.

Reply:
I updated here and the other log messages to combine them. I'm going to make a follow-up PR to make the equivalent change in the command log message suggested string representations